app/notifications/rest.py

```diff
@@ -181,9 +181,10 @@ def _service_allowed_to_send_to(notification, service):
         if api_user.key_type == KEY_TYPE_TEAM:
             message = 'Can’t send to this recipient using a team-only API key'
```

Comment:
You could link to a documentation page as well here

Reply:
linked added for team api